### PR TITLE
(WIP) - convert SHOW TAG VALUES to distributed query

### PR DESCRIPTION
**Pull request by dgnorton**

### CODE REVIEW

File: tsdb/show_tag_values.go [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+func NewShowTagValuesExecutor(stmt *influxql.ShowTagValuesStatement, mappers []Mapper, chunkSize int) *ShowTagValuesExecutor {
+	return &ShowTagValuesExecutor{
+		stmt:      stmt,
+		mappers:   mappers,
+		chunkSize: chunkSize,
+	}
+}
+
+// Execute begins execution of the query and returns a channel to receive rows.
+func (e *ShowTagValuesExecutor) Execute() <-chan *models.Row {
```
Comment (otoolep) [inline tsdb/show_tag_values.go]: We need templates! :-)

`influxdb::Executor<TagValues>`

File: tsdb/show_tag_values.go [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+		sort.Sort(whereMs)
+
+		measurements = measurements.intersect(whereMs)
+	}
+
+	// Create a channel to send tag values on.
+	ch := make(chan TagsValues)
+	// Start a goroutine to send the names over the channel as needed.
+	go func() {
```
Comment (otoolep) [inline tsdb/show_tag_values.go]: Why do it this way? Could you not just iterate over the measurements in `NextChunk()`?

**Comment by otoolep** [comment]
Should we expect some tests? If this is WIP, makes sense not so far.

**Comment by otoolep** [comment]
@dgnorton -- is this still WIP?

**Comment by otoolep** [comment]
@kostasb hit this, I think.

**Comment by jwilder** [comment]
@dgnorton Should this be closed?

**Pull request closed without merging**